orko-ui/src/containers/OrdersContainer.js

```diff
@@ -15,40 +15,61 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import React from "react"
-import Section from "../components/primitives/Section"
-import Tab from "../components/primitives/Tab"
-import OpenOrdersContainer from "./OpenOrdersContainer"
-import UserTradeHistoryContainer from "./UserTradeHistoryContainer"
-import GetPageVisibility from "../components/GetPageVisibility"
-import RenderIf from "../components/RenderIf"
+import React from 'react'
```

Comment:
Would you mind formatting your changes with Prettier?

https://plugins.jetbrains.com/plugin/10456-prettier

Makes diffs so much easier!